### ISSUE TITLE
novm on archlinux

### ISSUE THREAD

**Issue by thypon** [google/novm#34]
```
> novm-import-kernel 
Usage: extract-vmlinux <kernel-image>
error: Command '['~/Workspace/novm/lib/novm/libexec/extract-vmlinux', '/boot/vmlinuz-3.18.5-1-ARCH']' returned non-zero exit status 2
```

Importing Linux Kernel Image on Archlinux yields an error; archlinux is a rolling release and breaks the standard linux name conventions.

For example the actual kernel image named `core\linux` (representing 3.18.5-1-ARCH kernel) is placed /boot/vmlinuz-linux with modules under /lib/modules/3.18.5-1-ARCH.

If copy vmlinuz-linux to the expected location the error message become:

```
> novm-import-kernel
[things]
error: [Errno 2] No such file or directory: '/boot/System.map-3.18.5-1-ARCH'
```

In archlinux the System.map file isn't available in /boot/System.map-${put here the version} but it is easily recoverable from a running kernel with

```
> sudo cp /proc/kallsyms /boot/System.map-`uname -r`
```

After these shims novm works.

**Comment by hamo** [comment]
Yes, arch does not follow some standards. novm-import-kernel is just a wrapper of novm mkkernel and novm getkernel.

You can call following commands:
$ fakeroot novm mkkernel --release RELEASE --vmlinux VMLINUX  --sysmap SYSMAP --setup SETUP
$ fakeroot novm --url PRE_COMMAND_OUTPUT
